themes/edx.org/lms/templates/header.html

```diff
@@ -144,12 +144,7 @@ <h2 class="course-header">
 </header>
 % if course:
 <!--[if lte IE 9]>
-<div class="ie-banner" aria-hidden="true">${Text(_('{begin_strong}Warning:{end_strong} Your browser is not fully supported. We strongly recommend using {chrome_link} or {ff_link}.')).format(
-    begin_strong=HTML('<strong>'),
-    end_strong=HTML('</strong>'),
-    chrome_link=HTML('<a href="https://www.google.com/chrome" target="_blank">Chrome</a>'),
-    ff_link=HTML('<a href="http://www.mozilla.org/firefox" target="_blank">Firefox</a>'),
-)}</div>
+<div class="ie-banner" aria-hidden="true">$(Text(_('{strong_open_tag}Warning:{strong_close_tag} Your browser is not fully supported. We strongly recommend using {chrome_link} or {ff_link}.').format(chrome_link=HTML('<a href="https://www.google.com/chrome" target="_blank">Chrome</a>'), ff_link=HTML('<a href="http://www.mozilla.org/firefox" target="_blank">Firefox</a>'), strong_open_tag=HTML('<strong>'),strong_close_tag=HTML('</strong>'),)}</div>
```

Comment:
This line is super long, can we get it back to being split across several lines for readability?